spec/index.md

```diff
@@ -219,6 +219,8 @@ The additional features provided by the Interface (KL27) via I2C are:
 | Read property disallowed                            | 0x36       |
 | Write property disallowed                           | 0x37       |
 | Write fail                                          | 0x38       |
+| Busy                                                | 0x39       |
```

Comment:
Can we add a description somewhere about what this is and how it should be used?

Reply:
This is covered in the "considerations section".